pkg/controller/controller.go

```diff
@@ -67,6 +73,9 @@ func AddToManager(m manager.Manager, deps Dependencies) error {
 		if a2, ok := a.(GetPodInjector); ok {
 			a2.InjectGetPod(deps.GetPod)
 		}
+		if a3, ok := a.(GetConfigMatchSetInjector); ok {
+			a3.InjectConfigMatchSet(deps.ConfigMatchSet)
```

Comment:
no need to create an `a3`, `a2` will work here as well. We just don't want to mistake the non-type-casted injector with the type-casted one.

Reply:
thanks, updated